src/constants.ts

```diff
@@ -29,3 +29,5 @@ export const CustomFieldsMap = {
 };
 
 export const IMPORTANCES = ["High", "Normal", "Low"];
+
+export const STATUSES = ["Active", "Completed", "Deferred", "Cancelled"];
```

Comment:
Should this be an `enum`

Reply:
Deleted. I didn’t notice after the refactoring that it wasn’t being used anywhere.